src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -655,9 +655,17 @@ private SyntaxToken RenameWithinToken(SyntaxToken oldToken, SyntaxToken newToken
                     return newToken;
                 }
 
-                if (_isRenamingInStrings && newToken.IsKind(SyntaxKind.StringLiteralToken))
+                if (_isRenamingInStrings)
                 {
-                    newToken = RenameInStringLiteral(oldToken, newToken, SyntaxFactory.Literal);
+                    if (newToken.IsKind(SyntaxKind.StringLiteralToken))
+                    {
+                        newToken = RenameInStringLiteral(oldToken, newToken, SyntaxFactory.Literal);
+                    }
+                    else if (newToken.IsKind(SyntaxKind.InterpolatedStringTextToken))
+                    {
+                        newToken = RenameInStringLiteral(oldToken, newToken, (leadingTrivia, text, value, trailingTrivia) =>
```

Comment:
VB has SyntaxFactory.InterpolatedStringTextToken but C# does not. This is an API bug, right? @mattwar / @Pilchie